tests/models/recipes/test_recipe.py

```diff
@@ -1,16 +1,21 @@
 from reciperadar.models.recipes import Recipe
 
 
-def test_recipe_from_doc(raw_recipe_hit):
+def test_recipe_from_doc(db_session, raw_recipe_hit):
     recipe = Recipe().from_doc(raw_recipe_hit['_source'])
+
+    # Commit recipe to the session; performs db object relationship resolution
```

Comment:
TODO: it would be nice to figure out whether there is a more robust way to achieve this relationship resolution.  At the moment requiring both the `db_session` fixture and a database commit is fragile; either of these could easily be dropped accidentally from future test cases.